subprojects/platform-jvm/src/main/java/org/gradle/jvm/tasks/Jar.java

```diff
@@ -92,6 +98,16 @@ public void execute(FileCopyDetails details) {
         });
     }
 
+    @Override
+    protected CopyAction createCopyAction() {
```

Comment:
Couldn’t we just add the method `createCopyAction(Comparator)` which is called from `createCopyAction()`?